torch/onnx/symbolic_helper.py

```diff
@@ -287,19 +292,21 @@ def wrapper(g, *args, **kwargs):
             if _zero_point is not None:
                 _zero_point = g.op("Constant", value_t=torch.tensor(_zero_point))
 
-            # some args may be optional, so the length may be smaller
-            assert len(arg_q_descriptors) >= len(args)
-            desc_args = tuple(zip(arg_q_descriptors[: len(args)], args))
+            # Support variable length arguments by marking unspecified ones as non-quantized
```

Comment:
I created #78775. This PR will depend on #78775 since we need the feature to add quantization support for `_interpolate` (which has variable length parameters)